src/ImageViewer.cpp

```diff
@@ -857,6 +859,27 @@ void ImageViewer::removeImage(shared_ptr<Image> image) {
     }
 }
 
+void ImageViewer::removeAllImages() {
+    if (mImages.empty())
+        return;
+
+    // Reset all focus as a workaround a crash caused by nanogui.
+    // TODO: Remove once a fix exists.
+    requestFocus();
+
+    for (long i = mImages.size() - 1; i >= 0; --i) {
```

Comment:
This would be more correct as `for (size_t i = mImages.size(); i > 0; --i) {` with a subtraction of `1` from `i` within the loop body.